tests/karate/docs/README.md

```diff
@@ -0,0 +1,172 @@
+          "description": '#(validRollCall.description)',
+        }
+      """
+    When organizer.publish(validCreateRollCall, lao.channel)
+    And json answer = organizer.getBackendResponse(validCreateRollCall)
+    Then match answer contains VALID_MESSAGE
+    And match organizer.receiveNoMoreResponses() == true
+```
+
+## First Setup
+
+### All
+Make sure that you have [maven](https://maven.apache.org/download.cgi) installed.
+The Karate plugin for IntelliJ is also recommended.
+
+### Backend
+
+To test a backend, you don't need more setup that what is needed to build that backend. Use the resources provided by
+those projects.
+
+### Android Front-end
+
+To test the Android Frontend, you need to have an Android emulator installed. The easiest way to achieve it is to
+install it through [Android Studio](https://developer.android.com/studio) :
+Go to `Tools -> AVD Manager` and create an emulator.
+
+Then you need to install Appium. You can install either the command line app with `npm install -g appium`, or
+the [Desktop App](https://github.com/appium/appium-desktop/releases/).
+
+Finally, you need to set the environment variable `ANDROID_HOME` (The previous name was`ANDROID_SDK_ROOT`
+and it still works)
+to your Android SDK installation. Find it by
+opening Android Studio and going to `Tools -> SDK Manager`. It stands next to `Android SDK Location`.
+
+If your Computer runs on Windows : we strongly advise that you do not use a VM or WSL. You will encounter problems you
+would not have otherwise, some of which might even be technically impossible to solve.
+
+### Web Front-end
+
+Make sure you have [Google Chrome](https://www.google.com/intl/en/chrome/) and [npm](https://nodejs.org/en/download/)
+installed.
+
+## Running the Tests
+
+### Backend
+
+Build the backend you want to test. Follow the steps described in the corresponding subproject.
+
+Simply run all the tests with:
+
+```
+mvn test -DargLine="-Dkarate.env=go"
+mvn test -DargLine="-Dkarate.env=scala"
+```
+
+Individual scenarios can be run with:
+
+```
+mvn test -Dkarate.options="--tags @scenarioTagName"
+```
+
+With the Karate plugin for IntelliJ, the full tests can also be run directly from inside IDE in the `BackEndTest` class.
+
+
+### Android Front-end
+
+Build the application by running `./gradlew assembleDebug` in the corresponding directory.
+
+Start the Android Emulator. Start Appium : if you use the GUI, delete the text in Host and Port and click on the start
+server button. If you use the terminal, run `appium`.
+
+With Android Bumblebee the emulator can either run in a tool window or a standalone window. (To have it in a standalone
+window, go to `File -> Settings -> Tools -> Emulator` and unselect `Launch in a tool window`).
+- Standalone window : \
+The emulator window name should match : `Android Emulator - avd:id` \
+Ex: `Android Emulator - Pixel_4_API_30:5554`
+- Tool window : open the `Extended Controls` (the 3 points above the emulator)
```

Comment:
> the 3 points above the emulator

This kind of thing might be clearer with a couple of screenshots.

Reply:
I don't have access to Android Studio right now, but I can do this at the end of the week.